install_ffmpeg.sh

```diff
@@ -24,10 +24,15 @@ if [ ! -e "$HOME/x264/x264" ]; then
 fi
 
 if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
-  git clone https://github.com/livepeer/FFmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
+  LIBTENSORFLOW_VERSION=2.3.0 \
```

Comment:
cc @jailuthra for thoughts on whether this should be always installed or whether it should be conditional similar to the approach that is being worked on for https://github.com/livepeer/go-livepeer/issues/1919.

Reply:
Yeah makes sense to keep it conditional on `BUILD_TAGS="experimental"`.

Also looks like this is not really needed to get the CI build+test scripts running (https://github.com/livepeer/lpms/tree/yf/adddnn) - So I'm fine if we comment it out altogether for now, and handle this later when we enable the unit tests and fix the build process.

Reply:
@oscar-davids What do you think about commenting this out for now and moving ahead for the merge?

Reply:
That makes sense. So we want to comment all related libtensorflow? Like this?

//&& curl -LO https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux- ...
//--enable-libtensorflow

Reply:
Yep that sounds right. Everything from L27-L30